components/CommentForm.jsx

```diff
@@ -36,7 +36,7 @@ return(
         <label htmlFor="comment">Comment:</label>
         <input type="text" id="comment" name="body" onChange={handleChange} value={formData.body} required/>
 
```

Comment:
Your form still has an input for a user to put a username when posting a comment. Now that you have a logged in user from context, you don't need this input and can instead send the logged in user's username to your api call instead 😄